react-frontend/src/components/Resources/fordesigners.js

```diff
@@ -16,99 +19,99 @@ function ForDesigners() {
     <ContentBlockContainer id="forDesigners">
       <Row>
         <Col sm={12}>
-          <Row>
+          <ResourceLinkRow>
```

Comment:
Just a thought on what to name this .... what about something like "NestedRow"? ... It doesn't really have anything to do with the fact that this is on the Resources page per se ...  it could be useful on any page where a row needs to be embedded within another row + column.

Reply:
My thoughts on that are to wait until we need it on multiple pages to rename it, thoughts?

Reply:
That's fine, youda boss 🙇‍♂️